@mol-view-stories/webapp/src/app/auth/page.tsx

```diff
@@ -29,8 +29,12 @@ export default function AuthPage() {
   const [message, setMessage] = useState('');
   const [error, setError] = useState<string | null>(null);
   const [timeoutReached, setTimeoutReached] = useState(false);
+  const [authProcessed, setAuthProcessed] = useState(false);
```

Comment:
This doesn't need to cause a re-render, so you can use `const authProcessed = useRef(false)` and then set/access it via `authProcessed.current`

Reply:
Feel free to merge and deploy when fixed.